ironfish/src/wallet/wallet.ts

```diff
@@ -973,6 +973,11 @@ export class Accounts {
 
       await this.db.removeAccount(account, tx)
       await this.db.removeHeadHash(account, tx)
+      await this.db.clearDecryptedNotes(account, tx)
+      await this.db.clearNullifierToNoteHash(account, tx)
+      await this.db.clearTransactions(account, tx)
+      await this.db.clearSequenceToNoteHash(account, tx)
+      await this.db.clearNonChainNoteHashes(account, tx)
```

Comment:
This is gonna hang my node for hours potentially. We should test this on the bank and the pool.

Reply:
Tested this on my pool account (~9k notes) and removed the account in 10 seconds

On my partially-scanned bank account (~4M notes) it took about 30 minutes

Reply:
This has been changed to an incremental cleanup in the background, so this is no longer a problem.